packages/deployment/package.json

```diff
@@ -4,13 +4,13 @@
   "description": "Deployment templates for cumulus",
   "scripts": {
     "test": "ava",
-    "test-coverage": "nyc ava",
+    "test-coverage": "nyc npm test",
```

Comment:
What's the difference between doing it this way and how it was before?

Reply:
Before, if we wanted to change the test command (from say `ava` to `ava --serial`, for example), we would have to change it in two places.  This way there's only one place that we define what "test" means.

Reply:
It's a minor thing, but this change DRYs up our `package.json`.

Reply:
![](https://media.giphy.com/media/d3mlE7uhX8KFgEmY/giphy.gif)